emission/integrationTests/docker-compose.yml

```diff
@@ -1,14 +1,13 @@
 version: "3"
 services:
   web-server:
-    build: ../../integrationTests
+    build: integrationTests
```

Comment:
I don't think this works if you are already in `integrationTests`

Reply:
Trying out using "context" keyword to build in the current directory, rather than moving the files.